modules/schema/src/test/java/org/apache/ignite/internal/schema/configuration/SchemaConfigurationConverterTest.java

```diff
@@ -332,32 +345,11 @@ private static Iterable<DefaultValueArg> generateTestArguments() {
 
         for (var entry : DEFAULT_VALUES_TO_TEST.entrySet()) {
             for (var defaultValue : entry.getValue()) {
-                if (skipValue(entry.getKey(), defaultValue)) {
-                    continue;
-                }
-
                 paramList.add(
                         new DefaultValueArg(specToType(entry.getKey()), adjust(defaultValue))
                 );
             }
         }
         return paramList;
     }
-
-    // TODO: IGNITE-17370 remove this
```

Comment:
should we close IGNITE-17370 at all?

Reply:
Currently the mentioned issue is still actual in master, hence we need to close this after current patch will be merged to master